.github/ISSUE_TEMPLATE/research.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Research
+about: For setting up and conducting research sessions and synthesizing findings
+title: ''
+labels: experience, research
+assignees: ''
+
+---
+
+## Overview
+
+[Outline what we are researching and why. What are the goals?]
+
+## Participants
+
+**Interview subject(s)**: [name/organization/etc]
+**Contact info**: [ ... ]
+
+## Background
+
+[Any info on participants/session goals/etc to share ... ]
+
+## To do
+
```

Comment:
- [ ] Do "homework" on the the population or individual that you will be meeting with; what are the previous calls with them; googling the organization they work for and any relevant information where their documentation might address some interview questions you have

Reply:
Added these as a checklist in the "background" section!